src/base/bittorrent/session.cpp

```diff
@@ -4492,6 +4504,20 @@ void Session::handleFileErrorAlert(const lt::file_error_alert *p)
     m_recentErroredTorrentsTimer->start();
 }
 
+void Session::handleReadPieceAlert(const lt::read_piece_alert *p)
+{
+    lt::error_code ec;
+    if (ec) {
```

Comment:
I think you should add an LogMsg() inside this block.

Reply:
Already a file error alert is getting logged. I don't think it's necessary.

Reply:
![Capture2](https://user-images.githubusercontent.com/47396879/75263680-84481f80-5818-11ea-830c-dddbc143a29b.PNG)